demo/demo.md

```diff
@@ -128,14 +73,44 @@ Depending on the element trigger that the popover element is assigned to, the sp
   <auro-button id="button10" slot="trigger">Popover w/additional space</auro-button>
 </auro-popover>
 
-<auro-popover for="button11" placement="bottom">
+<auro-popover for="button11" placement="bottom" addSpace>
   This popover uses the default spacing<br>between the popover and the trigger.
   <auro-button secondary id="button11" slot="trigger">Popover w/o additional space</auro-button>
 </auro-popover>
 
 ```
 </auro-accordion>
 
+## Remove space around popover
+
+Sometimes you just need less space. For these instances, use the `removeSpace` attribute.
+
+<div class="exampleWrapper">
+  <!-- The slot=trigger attribute is bound directly to the auro-icon element  -->
+  <auro-popover for="plugIcon" removeSpace>
+    Notice this popover is a little<br>closer to the trigger.
+    <auro-icon id="plugIcon" category="in-flight" name="plug" slot="trigger" tabindex="0"></auro-icon>
+  </auro-popover>
+</div>
+
+<auro-accordion lowProfile justifyRight>
+  <span slot="trigger">See code</span>
+
+```html
+<!-- The slot=trigger attribute is bound directly to the auro-icon element  -->
```

Comment:
`addSpace` demo has the default shown next to it to highlight the change. Can we do that here as well?

Reply:
Mor typoes.